skimage/morphology/tests/test_reconstruction.py

```diff
@@ -139,3 +139,10 @@ def test_offset_not_none():
     assert_array_almost_equal(
         reconstruction(seed, mask, method='dilation',
                        selem=np.ones(3), offset=np.array([0])), expected)
+
+
+def test_deprecated_import():
+    with testing.expected_warnings(
+        ["Importing from skimage.morphology.greyreconstruct is deprecated."]
+    ):
```

Comment:
```suggestion
    msg = "Importing from skimage.morphology.greyreconstruct is deprecated."
    with testing.expected_warnings([msg]):
```
It's one line shorter and it's the way you did in `skimage/morphology/tests/test_gray.py` ;)